.travis.yml

```diff
@@ -6,10 +6,10 @@ language: csharp
 # Run the build script
 # call RALint to check for YAML errors
 script:
+ - make check
```

Comment:
make check invokes the Utility, which hasn't been built yet. Those two targets (make and check) should probably depend on the `utility` target.